packages/strapi-provider-upload-cloudinary/lib/index.js

```diff
@@ -21,6 +21,17 @@ module.exports = {
               if (err) {
                 return reject(err);
               }
+
+              if (image.resource_type === 'video') {
+                file.previewUrl = cloudinary.url(`${image.public_id}.gif`, {
+                  video_sampling: 6,
+                  delay: 200,
+                  width: 250,
+                  crop: 'scale',
+                  resource_type: 'video',
+                });
+              }
```

Comment:
I believe it works but I don't understand how cloudinary knows from which video you want a preview. Is it the name of the file that is unique and the extension doesn't matter ?

Reply:
exactly :)

Reply:
ok, thanks :)